packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -608,7 +607,18 @@ export class Form<N extends CollectionNameString> extends Component<SmartFormPro
         ) {
           continue;
         }
-        this.setState(getInitialStateFromProps(nextProps));
+        if (prop === "prefilledProps") {
```

Comment:
This change address the `TODO` on line 40 of this file. When changing prefilled props we don't want to reinitialize the editor - just update the relevant fields of the document.